File: problemas/matematica/3049/3049.scala
object Main {
    def main(args: Array[String]) {
        val B = io.StdIn.readLine().toInt
        val T = io.StdIn.readLine().toInt

        if (B + T > 160) {
            println(1)
        } else if (B + T < 160) {
            println(2)
        } else {
            println(0)
        }
    }
}

File: problemas/matematica/3049/3049.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let b: i32 = input.trim().parse().unwrap();

    input = String::new();
    io::stdin().read_line(&mut input);
    let t: i32 = input.trim().parse().unwrap();

    if (b + t > 160) {
        println!("1");
    } else if (b + t < 160) {
        println!("2");
    } else {
        println!("0");
    }
}
